commons/all/src/main/java/org/infinispan/commons/util/SslContextFactory.java

```diff
@@ -26,6 +26,17 @@
  * @since 5.3
  */
 public class SslContextFactory {
+   public enum SslContextName {
```

Comment:
As this would only be used by tests, I'd remove it from here

Reply:
Done